spark/src/main/scala/geotrellis/spark/ingest/CellGridPrototype.scala

```diff
@@ -0,0 +1,8 @@
+package geotrellis.spark.ingest
+
+import geotrellis.raster.CellType
+
+trait CellGridPrototype[T] {
+  def prototype(cols: Int, rows: Int): T
+  def prototype(cellType: CellType, cols: Int, rows: Int): T
+}
```

Comment:
What is going on here? Looks like something from raster but we have those `TilePrototypeMethods`.